.gitattributes

```diff
@@ -0,0 +1,3 @@
+# do not modify line endings of our hat test golden files
+packages/cursorless-engine/src/test/fixtures/hat-stats/*.golden -crlf
```

Comment:
I thought the attribute is called `eol=lf`, or `-text` for "don't mangle this at all"

Reply:
Oh no not this again 😅. What's the issue you're trying to solve here @josharian?

Reply:
It’s solved now, thanks to @auscompgeek .

The issue was that git was adding CR to the golden files on checkout, so the test that the golden files were unmodified was failing.

Reply:
I would prefer to avoid gitattributes if at all possible. If people want to check things out as CRLF, I don't see why we should get in the way of that. The last time we ran into this problem, we solved it by normalizing when we read the gold file. See https://github.com/cursorless-dev/cursorless/blob/5789094879106b0417f73b21d1c872d3cc6d618a/packages/cursorless-vscode-e2e/src/suite/testCaseRecorder.vscode.test.ts#L169-L171

Reply:
OK. If you look at the commit history, I’ve also implemented that, when you gitattributes originally did not work. Although to avoid needless diffs, we also have to normalize at writing time as well…sigh.